README.md

```diff
@@ -111,6 +111,8 @@
 - [TiDB Best Practices](https://pingcap.github.io/blog/2017/07/24/tidbbestpractice/)
 - [Releases](releases/rn.md)
 - [TiDB Roadmap](https://github.com/pingcap/docs/blob/master/ROADMAP.md)
++ Use Cases
+  - [Yiguo.com](https://www.datanami.com/2018/02/22/hybrid-database-capturing-perishable-insights-yiguo/)
```

Comment:
Please change the user name to the title of the news because the clients from abroad might have no idea who the user is.